dev-tools/run.sh

```diff
@@ -46,6 +43,7 @@ if nc -w1 localhost 5432; then
     ./dev-tools/migrate.sh
 
     # Start Integreat CMS
+    pipenv run integreat-cms-cli tailwind start &\
```

Comment:
```suggestion
    pipenv run integreat-cms-cli tailwind start &
```
I don't understand the backslash at the end of the line, is this a mac os thing?

Reply:
The backslash escapes the newline. It would evaluate to this form: `pipenv run integreat-cms-cli tailwind start & pipenv run integreat-cms-cli runserver localhost:8000`.
I added it because the script will run in the background and watch for configuration/scss changes which should end when the run.sh script is terminated. But as I now learned a terminated shell script will kill it's child processes. 👍🏽

Reply:
Interesting, I didn't know that it's possible to write subsequent commands after `&`...